pkg/engine2/testdata/vpc_import_to_lambda.expect.yaml

```diff
@@ -31,7 +31,7 @@ resources:
             - aws:security_group:vpc:lambda_function-security_group
         Subnets:
             - aws:subnet:vpc:subnet1
-            - aws:subnet:subnet2
+            - aws:subnet:vpc:subnet2
```

Comment:
Was this fixed by the `&& setConstraint.Operator == ""` to the vertex property?

Reply:
yup